README.md

```diff
@@ -689,6 +689,7 @@ Table of Contents
 
    * [conveyor.cloud](https://conveyor.cloud/) — Visual Studio extension to expose IIS Express to the local network or over a tunnel to a public URL.
    * [Hamachi](https://www.vpn.net/) — LogMeIn Hamachi is a hosted VPN service that lets you securely extend LAN-like networks to distributed teams with free plan allows unlimited networks with up to 5 peoples
+   * [Radmin VPN](https://www.radmin-vpn.com/) - Connect multiple computers together via a VPN enabling LAN-like networks. Unlimited peers. (Hamachi alternative)
```

Comment:
Whats the free tier of this? I do not see any free tier on their site

Reply:
There is no paid option. It's free.

Reply:
Hmm, think I put that comment on the wrong PR :) good to go, thank you!